lib/ContentHelpers.php

```diff
@@ -18,11 +31,17 @@ public static function getCommonDetailFields(Entry $entry, $status, $locale)
             'dateCreated' => $entry->dateCreated,
             'dateUpdated' => $entry->dateUpdated,
             'availableLanguages' => EntryHelpers::getAvailableLanguages($entry->id, $locale),
+            // @TODO: Is url used anywhere?
+            'url' => $entry->url,
+            // @TODO: Some older pages use path instead of linkUrl in templates, update these uses and then remove this
+            'path' => $entry->uri,
             'linkUrl' => $entry->externalUrl ? $entry->externalUrl : EntryHelpers::uriForLocale($entry->uri, $locale),
             'title' => $entry->title,
-            'trailText' => $entry->trailText ?? null,
+            'trailText' => $entry->trailText ?? $entry->title,
```

Comment:
Is this right? Would it start showing double titles in places where we don't have trailtext?

Reply:
Not sure I follow, why would it show double titles?

Reply:
Instances like this:

![image](https://user-images.githubusercontent.com/394376/49454318-cc76be80-f7dc-11e8-8d3c-bea13e2ad595.png)

That widget will render a card like this (if that element doesn't have trailtext):

![image](https://user-images.githubusercontent.com/394376/49454380-f203c800-f7dc-11e8-98ef-195ba9d1d0d8.png)

I think it's better here to reflect the fact that no trailtext was provided (and let the template handle that, eg. some components might choose to use the title instead, but others won't).

Reply:
Interesting, that suggests we're using "trailText" to mean two different things. In some cases trail text means "alternate headline to use in place of the title on listings" and in others meaning "short line of description to use in *addition* to the title on listings". I'd rather standardise what we use this field for (but appreciate that might be for a later PR)

Reply:
👍

Reply:
i.e. we got it right on the new updates section and should be using the `summary` here instead. Will roll this back for now though and make a ticket to review how we use this field more generally.

Reply:
Will roll this back here and raise a ticket in the backlog to sort this out more generally.